charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -126,6 +126,7 @@ node:
   kubeletPath: /var/lib/kubelet
   logLevel: 2
   priorityClassName:
+  affinity: {}
```

Comment:
Adding this here won't do anything without making changes to use it in the `node.yaml` and `node-windows.yaml` templates.

Reply:
@krmichel what do you need me to add ? just affinity fields ?

Reply:
Something like:
```
      {{- with .Values.node.affinity }}
      affinity:
        {{- toYaml . | nindent 8 }}
      {{- end }}
```